meetings/2021-07-21.md

```diff
@@ -50,7 +50,7 @@ But we need to be able to follow your nose through resources would be pretty pro
 
 Matthieu: There are two problems here:
 1. how does an agent know all the resources it has access to
-2. how does one knows which credentials to present to access a resource
+2. how does an agent know which credentials to present to access a resource
```

Comment:
```suggestion
1. how does an agent know all the resources it has access to?
2. how does an agent know which credentials to present to access a resource?
```

Reply:
I did not answer question 1. "how does an agent know all the resources it has access to?". Hmm...